bin/impl/load-env.sh

```diff
@@ -89,3 +89,11 @@ fi
 : ${ACCUMULO_USER:?"ACCUMULO_USER is not set in env.sh"}
 : ${ACCUMULO_PASSWORD:?"ACCUMULO_PASSWORD is not set in env.sh"}
 : ${ACCUMULO_TABLE:?"ACCUMULO_TABLE is not set in env.sh"}
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  export MD5=md5
```

Comment:
The `export` is needed.  Without `export`, the vars are available to `fluo-dev` but won't be to any process started by `fluo-dev`.  Adding export makes the var available to subprocesses.  I tested it to verify.  Also, see http://stackoverflow.com/questions/1158091/defining-a-variable-with-or-without-export